src/views/partials/card.handlebars

```diff
@@ -9,30 +9,30 @@
         </div>
         <div class="members-description">
             <div class="display-members">
-                <h2>Members</h2>
+                <h1>Members</h1>
                 {{#each card.members}}
                     {{#if this.assigned}}
                         <label>
                             <input type="checkbox" class="assigned" name="members[]" value="{{this._id}}" checked>
-                            <p class="card-member" style="background-color: {{this.color}};">{{this.initials}}</p>
+                            <p class="card-member">{{this.initials}}</p>
                         </label>
                     {{else}}
                         <label>
                             <input type="checkbox" class="unassigned" name="members[]" value="{{this._id}}">
-                            <p class="card-member" style="background-color: {{this.color}};">{{this.initials}}</p>
+                            <p class="card-member">{{this.initials}}</p>
```

Comment:
why are you getting rid of these background colors?

Reply:
It caused some contrast problems

Reply:
![image](https://user-images.githubusercontent.com/51027502/118056393-dd39a000-b357-11eb-870d-59f42436894e.png)
I actually see contrast problem on these ones, but not the ones in the card, can you try making them bigger as opposed to removing the color entirely as that is part of having users have colors

Reply:
I made them bigger but I dont remember Hill's login to test it, p.s I made like 35 boards on another account to try to get contrast errors and I couldnt so thats a rip

Reply:
It's in the slack

Reply:
Ok I checked and the larger ones didn't give any issues

Reply:
Nice! I'm not on my PC right now but I'll run the changes when I get on it in a few and make sure I have no other comments before I merge

Reply:
so the members in the card still dont have their color
![image](https://user-images.githubusercontent.com/51027502/118153105-7feb3080-b3e3-11eb-944e-c55866cee768.png)
can we get them to have their color back as this does not look good imo

Reply:
if using their individual bkgd color along with white text here still does not work, I would like at least to have some sort of border around each thing to separate the initials from each other because they look clumped here. But try using the individual colors first.

Reply:
Excellent work 🥇